### DISCUSSION TITLE
Transform a point(x,y) into a public key for Solana and Bitcoin

### DISCUSSION THREAD

**Discussion by cesariomartins** [Web3Auth/Web3Auth#1189] [Help]
Hi there, is there an example to transform tkey.metadata.pubKey (point object) into a bitcoin and solana public address?

Many tks

**Comment by shahbaz17** [comment]
Hey @cesariomartins 

For bitcoin, there is a way. You can use the standard library with the tKey to derive the address from privateKey.
